### ISSUE TITLE
GPG summary signatures not found

### ISSUE THREAD

**Issue by garrett** [hbons/org.sparkleshare.SparkleShare#2]
After doing the following:

``` bash
wget https://github.com/hbons/org.sparkleshare.SparkleShare/raw/master/sparkleshare.gpg
xdg-app --user remote-add --gpg-import=sparkleshare.gpg sparkleshare-releases http://releases.sparkleshare.org
xdg-app --user install sparkleshare-releases org.sparkleshare.SparkleShare
```

I get the message:

```
error: GPG verification enabled, but no summary signatures found (use gpg-verify-summary=false in remote config to disable)
```

I tried removing `~/.local/share/xdg-app/` and starting again, but got the same message.

**Comment by thet** [comment]
Same for me, but instead of xdg-app I get this error with flatpak.

**Comment by jstebbins** [comment]
I stumbled on this problem report while trying to solve my own problem.  I was getting this same error from a repo I created.  I would get it even when just trying to list the repo (flatpak remote-ls repo). I couldn't find any answers, but finally worked it out myself.  Thought I would share in case the issue here is related.

I was creating the repository directly using ostree and importing objects into it with flatpak build-import-bundle.  The bundles were signed when originally created.  But it seems that just importing signed bundles is not sufficient. In order to fix the error, I had to also run   

> ostree summary --repo=repo-dir --gpg-sign=key-id -u

So maybe the sparkleshare release repo has the same problem.
